crypto/cryptlib.c

```diff
@@ -167,22 +167,6 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_list ap;
     TCHAR buf[256];
     const TCHAR *fmt;
-# ifdef STD_ERROR_HANDLE        /* what a dirty trick! */
```

Comment:
-1. "What a dirty trick" refers to ... Windows CE. I mean if not for Windows CE, then this #ifdef wouldn't exits. And I mean #ifdef itself[!], not code that it surrounds. In other words code it surrounds is normally used.